src/util.ts

```diff
@@ -302,3 +302,21 @@ export function thisExtension() {
 export function thisExtensionPath(): string { return thisExtension().extensionPath; }
 
 export function dropNulls<T>(items: (T|null)[]): T[] { return items.filter(item => item !== null) as T[]; }
+
+export enum Ordering {
```

Comment:
Won't it be better to follow the common practice, adopted even for spaceship operator? (-1 for less)

Reply:
Spaceship operator also uses [a strong type](http://en.cppreference.com/w/cpp/utility/compare/strong_ordering).